src/content/h3/guides/shaders/pbr-shader/readme.md

```diff
@@ -137,7 +141,22 @@ Conversion Steps
 
 To make the texture importing more streamlined, end your texture file name with the {% key "_zbump" /%} flag to have the normal map import in the correct format.
 
-# Step 4: Bitmap Settings
+# Creating New Textures with Adobe Substance Designer
+
+If you have Adobe Substance Designer, you can easily export your new textures with this graph preset made by Grim! You can download it [here](substance_h3ppbr_presets.zip).
+
+* ![](https://user-images.githubusercontent.com/8784093/199033006-75124bb9-32cc-4328-940a-fd2db642c327.png "Substance Template")
```

Comment:
This doesn't need to be a bulleted list (`*`) as there's no list of things here, just an image.